pkg/webui/console/views/device-overview/index.js

```diff
@@ -146,6 +147,28 @@ class DeviceOverview extends React.Component {
     )
   }
 
+  get deviceMap () {
+    const {
+      ids,
+      description,
+      locations,
+    } = this.props.device
+
+    const markers = locations
+      ? [
+        {
+          name: description,
+          position: locations.user,
+        },
+      ] : false
+
+    if (markers) {
```

Comment:
When no location is present, I would like to see a grayed out, disabled version of the map, together with a message `No location information available`.